app/graphql/mutations/users/update_user_details_mutation.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  module Users
+    class UpdateUserDetailsMutation < Mutations::BaseMutation
+      description 'Updates a User with the access_token'
+      null true
+      argument :accessToken, String, required: false
+      argument :githubUsername, String, required: false
+      argument :githubOrg, String, required: false
+      field :errors, [String], null: true
+      def resolve(data)
+        current_user = context[:current_user]
+        if data[:github_org]
+          current_user.organization_id = Organization.find_or_create_by(name: data[:github_org]).id
+        end
+        current_user.update(data.slice(:access_token, :github_username))
```

Comment:
@hpjaj maybe you know a more idomatic rails way to handle this?

Reply:
Nah, this works.  Can we pls use `update!` so that an error is thrown, should there be an issue persisting the data.  And then it will be rescued and surfaced to the FE caller.

Here is an example of the that:

https://github.com/oddballio/magnifier/blob/master/app/graphql/mutations/statistics/update_statistic_mutation.rb#L29-L30